### DISCUSSION TITLE
Randomized TTL upon write cache

### DISCUSSION THREAD

**Discussion by grhamuda** [redisson/redisson#6123] [Q&A] [answered]
Hi All,
Can we randomized TTL upon write in redisson?
AFAIK in spring cache manager, we are able to overwrite `RedisCacheWriter` to randomized TTL when writing to cache
Thank you !

**Comment by mrniko** [comment, marked as answer]
Hi,

I don't any method with random ttl in RedisCacheWriter. Can you share the code example?

**Reply by grhamuda** [reply]
hi @mrniko, i've created this as solution for randomized TTL
lmk if you have any thought regarding this implementation
https://github.com/grhamuda/CustomRedissonCacheManager/blob/master/src/main/java/com/example/app/cache/manager/RedissonCache.java
